chemprop/data/utils.py

```diff
@@ -8,22 +9,33 @@
 
 from rdkit import Chem
 import numpy as np
+import pandas as pd
 from tqdm import tqdm
 
 from .data import MoleculeDatapoint, MoleculeDataset, make_mols
 from .scaffold import log_scaffold_stats, scaffold_split
 from chemprop.args import PredictArgs, TrainArgs
-from chemprop.features import load_features, load_valid_atom_or_bond_features, is_mol
+from chemprop.features import load_features, load_valid_atom_or_bond_features, is_explicit_h, is_adding_hs, is_mol
+from chemprop.rdkit import make_mol
+
+# Increase maximum size of field in the csv processing
+csv.field_size_limit(sys.maxsize)
 
 def get_header(path: str) -> List[str]:
     """
-    Returns the header of a data CSV file.
+    Returns the header of a data CSV or PICKLE file.
 
-    :param path: Path to a CSV file.
+    :param path: Path to a CSV or PICKLE file.
     :return: A list of strings containing the strings in the comma-separated header.
     """
-    with open(path) as f:
-        header = next(csv.reader(f))
+    extension = os.path.splitext(path)[1]
+
+    if extension in ['.pkl', '.pckl', '.pickle']:
+        df = pd.read_pickle(path)
+        header = df.columns.tolist()
```

Comment:
This function (maybe others too) in the data utils file could use unit tests showing that they can handle pkl inputs too.